coremltools/test/neural_network/test_neural_networks.py

```diff
@@ -243,6 +243,35 @@ def test_classifier_with_string_label_list(self):
             predicted_feature_name='classLabel',
             class_labels=['1', '2', '3', '4', '5', '6', '7', '8', '9', '10'])
 
+    def test_classifier_without_class_labels(self):
+        coremltools.converters.tensorflow.convert(
+            self.frozen_graph_file,
+            mlmodel_path=self.converted_coreml_file,
+            input_name_shape_dict={'input': [1, 224, 224, 3]},
+            image_input_names=['input'],
+            output_feature_names=['Softmax'])
+
+    def test_classifier_nhwc(self):
+        # Test manually specifying the image format. The converter would have
+        # detected NHWC.
+        coremltools.converters.tensorflow.convert(
+            self.frozen_graph_file,
+            mlmodel_path=self.converted_coreml_file,
+            input_name_shape_dict={'input': [1, 224, 224, 3]},
+            image_input_names=['input'],
+            output_feature_names=['Softmax'],
+            image_format='NHWC')
+
+    def test_classifier_nchw(self):
+        # Expect failure - input dimensions are incompatible with NCHW
+        with self.assertRaises(ValueError) as e:
+            coremltools.converters.tensorflow.convert(
+                self.frozen_graph_file,
+                mlmodel_path=self.converted_coreml_file,
+                input_name_shape_dict={'input': [1, 224, 224, 3]},
+                image_input_names=['input'],
+                output_feature_names=['Softmax'],
+                image_format='NCHW')
```

Comment:
can we make these tests more tight.
For instance 
`mlmodel = coremltools.converters.tensorflow.convert()`
`spec = mlmodel.get_spec()` and then assert of something that we expect as a result if this change
like `spec.neuralNetwork.layers[0].WhichOneOf('layer') == 'transpose'` etc

Reply:
Hey Aseem, after merging from master, it looks like there are no longer layers that mention the input format. This broke the test that I had. Given that the model is "implicitly" 224x224x3 and no transposes are occurring as far as I can tell, I'm considering removing this extra check. Let me know what you think.

Reply:
My hypothesis is that the missing transpose layer that I was asserting in my test was redundant and eliminated with e14a346c5c5f3067584df904fd5772dfdbf7e7d1

Reply:
Right

Reply:
Whats the use case then to add this flag`image_format`. When would a user use it? 
Also, I now realize it would be better if instead of calling the argument `image_format` we make it more explicit to something like `tf_image_format`? In coreml the image format is always NCHW, but in TF it can change